services/README.md

```diff
@@ -0,0 +1,5 @@
+# Services
+
```

Comment:
It would be worth saying a sentence about what the Services directory is for (and perhaps also what it's not for). Is that clear at the moment? I suspect it might not be, otherwise it would be easier to determine whether the proxy should live hear or not?

Reply:
Yes, I think that's a fine sentence and more importantly it reflects reality.

I wouldn't bother with the ticket, unless you're confident you'll come back to it. I suspect it will resolve itself in future as more changes are made.

Reply:
thanks, could you approve this PR then @lucyb & we can forget about this again for another year or three...?